libraries/AP_L1_Control/AP_L1_Control.cpp

```diff
@@ -179,7 +179,7 @@ float AP_L1_Control::loiter_radius(const float radius) const
 
 bool AP_L1_Control::reached_loiter_target(void)
```

Comment:
There are quite a few callers of this method. That would need testing. It would make it easier to review if the stale check was done in the `LoiterAltQLand` mode.